docker/package/wizard_structure.py

```diff
@@ -416,7 +433,7 @@ def pprint_options(self):
                     i += 1
         elif self.options and isinstance(self.options, dict):
             index_len = len(str(len(self.options)))
-            str_format = f"{{:{index_len}}}. {{:<26}}  {{}}"
+            str_format = f"{{:{index_len}}}. {{:<43}}  {{}}"
```

Comment:
Now some `tezos-setup-wizard` steps look a bit weird:
```
Do you want to set up baking or to run the standalone node?
1. baking (default)                             Set up and start all services for baking: tezos-node and
                               tezos-baker.
2. node                                         Only bootstrap and run the Tezos node.
```
```
Which history mode do you want your node to run in?
1. rolling (default)                            Store a minimal rolling window of chain data, lightest
                               option
2. full                                         Store enough chain data to reconstruct the complete chain
                               state
3. archive                                      Store all the chain data, very storage-demanding


```
I'd like to avoid hardcoding `43` here. Perhaps you can use the length of the largest option plus some constant instead